addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -2644,33 +2720,26 @@ export class OdooEditor extends EventTarget {
         if (anchorNode && !ancestors(anchorNode).includes(this.editable)) {
             return false;
         }
+        const selectSingleCell  = this._currentKeyPress?.isShiftKey && this._wasTableSelected;
+        delete this._wasTableSelected;
```

Comment:
Why? What does this fix? If this was a seperate bug can we have this in a different commit?

Reply:
This one handles the single cells selection when we try to shrink the selection from two selected cells. I think this one should be in second commit :)